opendcs-web-client/src/test/java/selenium/LoginTest.java

```diff
@@ -0,0 +1,105 @@
+package selenium;
+
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.firefox.FirefoxDriver;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+//@Disabled("For demonstration purposes currently.")
```

Comment:
Should be disabled

Reply:
Sorry, I meant to have that disabled.  Done.